src/test/java/com/targomo/client/api/request/config/RequestConfiguratorTest.java

```diff
@@ -10,6 +10,7 @@
 import com.targomo.client.api.geo.DefaultTargetCoordinate;
 import org.apache.commons.io.IOUtils;
 import org.boon.Maps;
+import org.boon.Sets;
```

Comment:
I would make that an own issue (with low priority?). There are multiple uses of boon in the code.
I'd still want to keep it in because boon has handy methods for sets and maps but I think you mean for json serialization/deserialization to just use jackson instead of mostly jackson, sometimes boon, right?